proxy/envoy/mixer.go

```diff
@@ -125,20 +150,70 @@ func mixerHTTPRouteConfig(role proxy.Node, service string) *FilterMixerConfig {
 		},
 		QuotaName: MixerRequestCount,
 	}
-	if len(service) > 0 {
-		r.MixerAttributes[AttrDestinationService] = service
+	v2 := &mccpb.MixerFilterConfig{
+		MixerAttributes: &mpb.Attributes{
+			Attributes: map[string]*mpb.Attributes_AttributeValue{
+				AttrDestinationIP:  {Value: &mpb.Attributes_AttributeValue_BytesValue{net.ParseIP(role.IPAddress)}},
+				AttrDestinationUID: {Value: &mpb.Attributes_AttributeValue_StringValue{"kubernetes://" + role.ID}},
```

Comment:
Can we use this opportunity to externalize or parameterize this bit? May be have some platform identity as a core attribute of the role struct (defaults to k8s). You might have to have a separate PR to change the role struct and all proxy agent invocation (eg k8s~...~..). Then this becomes usable in non k8s environments as well.

Reply:
+1 - I'll follow-up in another PR.